work-item-link-type-blackbox_test.go

```diff
@@ -150,6 +152,19 @@ func (s *workItemLinkTypeSuite) TestCreateAndDeleteWorkItemLinkType() {
 	_ = test.DeleteWorkItemLinkTypeOK(s.T(), nil, nil, s.linkTypeCtrl, *workItemLinkType.Data.ID)
 }
 
+// TestCreateWorkItemLinkType tests if we can create the "test-epic-relates-userstory" work item link type
+func (s *workItemLinkTypeSuite) TestCreateAndDeleteWorkItemEpicUserStoryLinkType() {
```

Comment:
@Ritsyy I agree with @baijum . You're not testing for example that your concret link type is included in the response. That makes it as generic as the other check. I think you can skip this test unless you can convince us why we need it. Fair enough?